src/components/custom/_signup-new.js

```diff
@@ -37,21 +38,23 @@ const Line = styled.div`
     background-color: var(--color-grey-7);
 `
 const StyledText = styled(Text)`
-    font-size: var(--text-size-xs);
+    font-size: ${(props) => props.fontSize || 'var(--text-size-xs)'};
     @media ${(props) => device.tabletL && props.notedBox} {
         width: 13rem;
     }
     @media ${device.tabletS} {
-        font-size: ${(props) => props.size};
+        font-size: ${(props) => props.tabletFontSize};
     }
 `
 const NoteBox = styled.div`
+    display: flex;
+    flex-direction: row;
     width: 100%;
-    padding: 1.6rem 1.6rem;
+    height: 7.7rem;
+    padding: 0.8rem 0.8rem;
     margin-top: 1.6rem;
-    border-radius: 0.6rem;
-    background-color: var(--color-grey-8);
-    border: solid 1px var(--color-grey-17);
+    border-radius: 0.4rem;
+    background-color: rgba(242, 243, 244, 0.56);
```

Comment:
```suggestion
    background-color: var(--color-grey-8);
    opacity: 0.56;
```

Reply:
I don't know why, but the result is not the same.